internal/errors.go

```diff
@@ -32,9 +32,9 @@ var knownErrors = map[string]error{
 	"context deadline exceeded": context.DeadlineExceeded,
 }
 
-// unwrapGRPCError removes the gRPC wrapper from the error and returns a known
+// UnwrapGRPCError removes the gRPC wrapper from the error and returns a known
 // error if possible, otherwise creates an internal error.
-func unwrapGRPCError(err error) error {
+func UnwrapGRPCError(err error) error {
```

Comment:
Now that I look at this, we shouldn't be reusing this in `pconduit`, as it can return an error coming from `pconnector`. Looks like we need separate utilities for `pconduit` and `pconnector` after all 😕

Reply:
@lovromazgon not sure I understand it. Wouldn't we unwrapping the error based on the request that errored (regardless of where this is actually coming from?)